demos/sample_azure_iot_pnp/sample_azure_iot_pnp.c

```diff
@@ -292,7 +292,7 @@ static void prvHandleProperties( AzureIoTHubClientPropertiesResponse_t * pxMessa
             break;
 
         default:
-            LogError( ( "Unknown property message: 0x%08x", pxMessage->xMessageType ) );
+            LogError( ( "Unknown property message: 0x%08x", ( uint16_t )pxMessage->xMessageType ) );
```

Comment:
Nitpick (I don't have a good solution outside of creating a lookup table): `xMessageType` is `typedef enum AzureIoTHubMessageType`.
Is there a guarantee that the size will be `uint16_t` and not even smaller? (e.g. if someone compiles with `-fshort-enums` potentially making this enum 1byte, would this still be a problem?)